apps/backend/src/libs/plugins/authorization-plugin/authorization-plugin.ts

```diff
@@ -36,9 +36,15 @@ const authorizationPlugin = fp<PluginOptions>((app, options, done) => {
 
 		try {
 			const {
-				payload: { userId },
+				payload: { exp, userId },
 			} = await token.decode(headerToken);
 
+			if ((exp as number) < Date.now()) {
```

Comment:
No need to check exp. If the token is expired, the plugin will throw an error and the user will be null anyway. On frontend, add a middleware to check if the error is UNAUTHORIZED

Reply:
token.decode does not throw an error when given an expired token